application/src/test/java/org/opentripplanner/model/TimetableTest.java

```diff
@@ -312,7 +312,7 @@ public void fixIncoherentTimes() {
     stopTimeUpdateBuilder.setStopSequence(1);
     stopTimeUpdateBuilder.setScheduleRelationship(StopTimeUpdate.ScheduleRelationship.SCHEDULED);
     var stopTimeEventBuilder = stopTimeUpdateBuilder.getArrivalBuilder();
-    stopTimeEventBuilder.setDelay(0);
+    stopTimeEventBuilder.setDelay(900);
```

Comment:
Will this test fail if you don't change it?

Reply:
The time will be valid after my code change because the bus can make up time during its journey.